custom-bundle.Dockerfile

```diff
@@ -13,6 +13,7 @@ ARG ANSIBLEEE_BUNDLE=quay.io/openstack-k8s-operators/openstack-ansibleee-operato
 ARG DATAPLANE_BUNDLE=quay.io/openstack-k8s-operators/dataplane-operator-bundle:latest
 ARG NOVA_BUNDLE=quay.io/openstack-k8s-operators/nova-operator-bundle:latest
 ARG IRONIC_BUNDLE=quay.io/openstack-k8s-operators/ironic-operator-bundle:latest
+ARG HORIZON_BUNDLE=quay.io/openstack-k8s-operators/horizon-operator-bundle:latest
```

Comment:
@stuggi with split we're going to have the `dependencies.yaml` [1], where a version can be specified for each bundle. 
In theory, grouping operators in an `openstack-storage` bundle means that the specific operator version (let's say manila for instance) is hidden and only visible/specified within the `openstack-storage` bundle.
The other key change is creating an index where >1 bundle are added/concatenated [2] (this will trigger within OLM the additional job).
Of course where you see `manila` in [1] [2] in a follow up will be a bundle that represents a group of operators (e.g. core, storage, extra).
@dprince to add more in case I'm overlooking something relevant.

[1] https://github.com/openstack-k8s-operators/openstack-operator/blob/master/dependencies.yaml#L5
[2] https://github.com/openstack-k8s-operators/openstack-operator/blob/master/.github/create_opm_index.sh#L13

Reply:
> @stuggi with split we're going to have the `dependencies.yaml` [1], where a version can be specified for each bundle. In theory, grouping operators in an `openstack-storage` bundle means that the specific operator version (let's say manila for instance) is hidden and only visible/specified within the `openstack-storage` bundle. The other key change is creating an index where >1 bundle are added/concatenated [2] (this will trigger within OLM the additional job). Of course where you see `manila` in [1] [2] in a follow up will be a bundle that represents a group of operators (e.g. core, storage, extra). @dprince to add more in case I'm overlooking something relevant.
> 
> [1] https://github.com/openstack-k8s-operators/openstack-operator/blob/master/dependencies.yaml#L5 [2] https://github.com/openstack-k8s-operators/openstack-operator/blob/master/.github/create_opm_index.sh#L13

@fmount ack, thanks for the pointer to the dependencies.yaml , I foregot about. Now I remember it was mentioned in the call we had last week. So if I got it correct, we'd have to update [2] to support pulling a custom bundle from e.g. my quay pointing to a specific version. not sure how this is done via the dependencies.yaml, but I think it is to early at this point and we'll see in the next weeks

Reply:
I've restored this file now that we're going to split it out.